Makefile

```diff
@@ -120,10 +120,14 @@ clean:
 	$(PYTHON) ./setup.py clean --all
 	rm -rf cover/ htmlcov/ docs/sphinx/_build/ build/ dist/
 
-rhsmcertd: $(DAEMONS_SRC_DIR)/rhsmcertd.c
+.PHONY: mkdir-bin
+mkdir-bin:
+	mkdir bin
```

Comment:
When `./bin` directory already exists, then the command fails.

Reply:
`mkdir -p` should help here.

Reply:
@mdellweg thanks for the suggestion! Will do.